### ISSUE TITLE
Test calculation of running variance in BatchNormedConv1d/Linear

### ISSUE THREAD

**Issue by julianhoever** [es-ude/elastic-ai.creator#322]
Test the calculation of running variance in the BatchNormedConv1d and BatchNormedLinear (see the following integration test which fails):

https://github.com/es-ude/elastic-ai.creator/blob/94269d2905c891dd69024e9c442fd42884083976/elasticai/creator/nn/fixed_point/conv1d/layer/batch_normed_conv1d_test.py#L52-L69
